OpenRA.Mods.Common/Effects/Bullet.cs

```diff
@@ -31,8 +31,8 @@ public class BulletInfo : IProjectileInfo
 		[Desc("Image to display.")]
 		public readonly string Image = null;
 
-		[Desc("Loop this sequence of Image while this projectile is moving.")]
-		[SequenceReference("Image")] public readonly string Sequence = "idle";
+		[Desc("Loop these sequences of Image while this projectile is moving.")]
+		[SequenceReference("Image")] public readonly string[] Sequences = { "idle" };
```

Comment:
While it seems none of the shipping mods use anything else than 'idle' (except the former shrapnel weapons), it would be better if there was an upgrade rule for this property rename anyway, for 3rd-party mods.